komodoenv/update.py

```diff
@@ -238,7 +206,9 @@ def check_same_distro(config):
     )
 
 
-def get_pkg_version(config, srcpath, package="komodoenv"):
+def get_pkg_version(
```

Comment:
I might be naive, but this doesn't look very robust to get the lib version. The way I usually do it is:
```sh
[komodo]/bin/pip show pandas --version | grep -i version
```

That would be given that we do have pip available :)

Reply:
We don't have pip available, no. I could perhaps call komodo's `python` to discover the version, but I don't know if that's a good thing.

I've used [PEP 376](https://www.python.org/dev/peps/pep-0376/) as basis for this work.